packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -149,8 +166,35 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
           /* @conditional-compile-remove(call-readiness) */
           onPermissionsTroubleshootingClick={onPermissionsTroubleshootingClick}
           errorBarProps={errorBarProps}
+          /* @conditional-compile-remove(call-readiness) */
+          callReadinessOptedIn={callReadinessOptedIn}
         />
       </Stack>
+
+      {
+        /* @conditional-compile-remove(call-readiness) */
+        mobileView && isDrawerShowing && callReadinessOptedIn && (
+          <_DrawerSurface onLightDismiss={onLightDismissTriggered} styles={drawerContainerStyles(99)}>
```

Comment:
99 should be defined somewhere as a constant, even just near the top of this file
```
const DRAWER_HIGH_Z_BAND = 99; // comment on why we choose an arbitrary 99
```